netaddr.go

```diff
@@ -234,6 +234,24 @@ func (ip IP) Unmap() IP {
 	return IP{v4Addr{a[12], a[13], a[14], a[15]}}
 }
 
+// IsLinkLocalUnicast reports whether ip is a link-local unicast address. If ip
```

Comment:
Start "If ip" sentence on its own line for readers reading source instead of godoc elsewhere.

Reply:
Done.